deploy/charts/alluxio/values.yaml

```diff
@@ -332,6 +332,20 @@ csi:
   # whether to launch CSI components
   enabled: false
 
+etcd:
+  auth:
+    client:
+      enableAuthentication: false
+    peer:
+      enableAuthentication: false
+    rbac:
+      allowNoneAuthentication: true
+      create: false
+    token:
+      enabled: false
+# Set num of etcd instances, it needs to be a odd number
```

Comment:
Do you mind add a brief introduction of the purpose of etcd component? 
Also does it make sense to start 3 etcd instances by default?

Reply:
i would assume start 3 instances would require at lease 3 physical nodes?

Reply:
No. It is possible to schedule multiple pods of the same statefulset onto one physical node.

Reply:
Offline discussed. 1 is fine.